js/components/portal-dashboard/feedback/question-feedback-textarea.tsx

```diff
@@ -31,7 +31,7 @@ export const QuestionFeedbackTextarea: React.FC<IProps> = (props) => {
   const updateFeedback = () => {
     if (answerId && textareaRef.current?.value !== undefined) {
       updateQuestionFeedback(answerId, {feedback: textareaRef.current?.value,
-                                        hasBeenReviewedForAnswerHash: answerHash(props.answer)});
+                                        hasBeenReviewedForAnswerHash: answerHash(answer)});
```

Comment:
Thanks for cleaning this up (props order, pulling answer from props)!  Sorry for rushing on this earlier today!

Reply:
No problem at all 👍